lib/MusicBrainz/Server/Controller/Artist.pm

```diff
@@ -426,6 +426,7 @@ sub recordings : Chained('load')
 
     $c->model('Recording')->load_meta(@$recordings);
 
+    my %release_group_appearances = $c->model('Recording')->appears_on($recordings, undef, 1);
```

Comment:
Might be a good idea to still set a limit (like 100 or something)? Looks like quite a few recordings have 300+ RGs and the most has 700 (though I didn't take into account duplicate name+type pairs). We could modify the `appears_on` query to join `release_group_meta` and sort by first_release_date columns if we do that.